trunk/SUAVE/Methods/Propulsion/Rotor_Wake/Fidelity_One/compute_PVW_inflow_velocities.py

```diff
@@ -0,0 +1,112 @@
+## @ingroup Methods-Aerodynamics-Common-Fidelity_Zero-Lift
+# compute_PVW_inflow_velocities.py
+#
+# Created:  Sep 2021, R. Erhard
+# Modified: Jan 2022, R. Erhard
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+from SUAVE.Core import Data
+from SUAVE.Methods.Propulsion.Rotor_Wake.Fidelity_One.compute_wake_induced_velocity import compute_wake_induced_velocity
+
+# package imports
+import numpy as np
+from scipy.interpolate import interp1d
+
+def compute_PVW_inflow_velocities( wake, prop, WD ):
```

Comment:
would prefer you spell out these acronyms

Reply:
changed to "compute_fidelity_one_inflow_velocities"